cartoframes/data/observatory/catalog/entity.py

```diff
@@ -110,9 +107,6 @@ def _get_print_id(self):
         return self.id
 
     def _download(self, file_path, credentials):
-        if not self._is_available_in('bq'):
```

Comment:
You can have access to DO, but the Dataset or Geography not been ready in BigQuery. How are we checking that?

Reply:
This comes from https://github.com/CartoDB/cartoframes/issues/1331#issuecomment-564115099. We are checking this with the `do/subscriptions` call after the download. However, I'll check this in the acceptance test.

Reply:
@arredond @andy-esch do you think we should keep this check? For the situation when a dataset is "not ready"?

Reply:
About the comment about the duplicated check, we were checking if the dataset/geo has a price which is done by the backend too.